### PR TITLE
fix egregiously incorrect sections

### PR DESCRIPTION
**Pull request by termie**
the sections had opposite descriptions, switched em

### CODE REVIEW

File: content/docs/ssh-keys/using-ssh-keys.md [hunk 1]
```diff
@@ -8,7 +7,14 @@ environment variable that will make use of a `SSH Key pair`.
 
 ### Add to known hosts pipeline step
 
-Using this step in combination with a `SSH Key pair` makes it possible to deploy your code.
+Depending on the container you are using, you may want to use this step so
+builds/deploys don't halt when you setup a connection with them:
```
Comment (mwhooker) [inline content/docs/ssh-keys/using-ssh-keys.md]: doesn't

File: content/docs/ssh-keys/using-ssh-keys.md [hunk 2]
```diff
@@ -22,14 +28,7 @@ deploy:
...
 
-Depending on the container you are using, you may want to use this step so
-builds/deploys don't halt when you setup a connection with them:
-
-```no-highlight
-The authenticity of host 'some-server.wercker.com (1.2.3.4)' can't be established.
-RSA key fingerprint is ff:ee:dd:cc:bb:aa:99:88:77:66:55:44:33:22:11:00.
-Are you sure you want to continue connecting (yes/no)?
-```
+Using this step in combination with a `SSH Key pair` makes it possible to deploy your code
```
Comment (mwhooker) [inline content/docs/ssh-keys/using-ssh-keys.md]: an `ssh key pair`

**Comment by mwhooker** [comment]
lgtm. might as well fix grammar while you're in there

File: content/docs/ssh-keys/using-ssh-keys.md (hunk 1 above)
Comment (termie) [inline content/docs/ssh-keys/using-ssh-keys.md]: don't looks fine, this is a plural

Comment (mwhooker) [inline content/docs/ssh-keys/using-ssh-keys.md]: oh yeah, wonder what I was thinking.

**Pull request merged**